scania-digital-design-system/tegel
# docs(dropdown): standardise order of controls

**Describe pull-request**  
Standardise the order of the controls. I chose to put "advanced" controls deafultOption and optionType at the bottom since most users wont bother with those.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-2037

**How to test**  
1. Go to Dropdown
2. Check the order of the controls.
